e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-page.spec.js

```diff
@@ -104,17 +105,61 @@ context('Your business - company details page - As an Exporter I want to enter m
   it('should display the trading name radios', () => {
     companyDetails.tradingName().contains(FIELDS[TRADING_NAME].LABEL);
     companyDetails.tradingNameYesRadio().should('exist');
+
+    companyDetails.tradingNameYesRadio().invoke('attr', 'aria-label').then((text) => {
+      expect(text.trim()).equal(`${FIELDS[TRADING_NAME].LABEL} yes radio`);
+    });
+
     companyDetails.tradingNameNoRadio().should('exist');
+
+    companyDetails.tradingNameNoRadio().invoke('attr', 'aria-label').then((text) => {
+      expect(text.trim()).equal(`${FIELDS[TRADING_NAME].LABEL} no radio`);
+    });
   });
 
   it('should display the trading address radios', () => {
     companyDetails.tradingAddress().contains(FIELDS[TRADING_ADDRESS].LABEL);
     companyDetails.tradingAddressYesRadio().should('exist');
+    companyDetails.tradingAddressYesRadio().invoke('attr', 'aria-label').then((text) => {
+      expect(text.trim()).equal(`${FIELDS[TRADING_ADDRESS].LABEL} yes radio`);
+    });
+
     companyDetails.tradingAddressNoRadio().should('exist');
+    companyDetails.tradingAddressNoRadio().invoke('attr', 'aria-label').then((text) => {
+      expect(text.trim()).equal(`${FIELDS[TRADING_ADDRESS].LABEL} no radio`);
+    });
   });
 
   it('should display the company website text area', () => {
-    companyDetails.companyWebsiteHeading().contains(FIELDS[WEBSITE].LABEL);
+    companyDetails.companyWebsiteHeading().invoke('text').then((text) => {
+      expect(text.trim()).equal(FIELDS[WEBSITE].LABEL);
+    });
     companyDetails.companyWebsite().should('exist');
+    companyDetails.companyWebsite().invoke('attr', 'aria-label').then((text) => {
+      expect(text.trim()).equal(FIELDS[WEBSITE].LABEL);
+    });
+  });
+
+  it('should display the phone number text area', () => {
+    companyDetails.phoneNumberHeading().invoke('text').then((text) => {
+      expect(text.trim()).equal(FIELDS[PHONE_NUMBER].LABEL);
+    });
+    companyDetails.phoneNumberHint().invoke('text').then((text) => {
+      expect(text.trim()).equal(FIELDS[PHONE_NUMBER].HINT);
+    });
+    companyDetails.phoneNumber().should('exist');
+    companyDetails.phoneNumber().invoke('attr', 'aria-label').then((text) => {
+      expect(text.trim()).equal(FIELDS[PHONE_NUMBER].LABEL);
+    });
+  });
+
+  it('should display the continue and save and go back button', () => {
+    continueButton().invoke('text').then((text) => {
+      expect(text.trim()).equal(BUTTONS.CONTINUE);
+    });
+
+    saveAndBackButton().invoke('text').then((text) => {
+      expect(text.trim()).equal(BUTTONS.SAVE_AND_BACK);
+    });
```

Comment:
can we use `submitButton` instead of `continueButton`?

Reply:
done